.github/workflows/reusable-operator-hub-release.yaml

```diff
@@ -0,0 +1,82 @@
+name: Reusable - Create operator hub pull request
+
+on:
+  workflow_call:
+    inputs:
+      org:
+        type: string
+        required: true
+      repo:
+        type: string
+        required: true
+      version:
+        type: string
+        required: true
+    secrets:
+      TEMPOOPERATORBOT_GITHUB_TOKEN:
+        required: true
+jobs:
+  create-operator-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Sync fork
+        env:
+          GH_TOKEN: ${{ secrets.TEMPOOPERATORBOT_GITHUB_TOKEN }}
+        run: |
+          # synchronizing the fork is fast, and avoids the need to fetch the full upstream repo
+          # (fetching the upstream repo with "--depth 1" would lead to "shallow update not allowed"
+          #  error when pushing back to the origin repo)
+          gh repo sync rubenvp8510/${{ inputs.repo }} \
```

Comment:
This command has your hardcoded username in it

Reply:
good catch! It was for testing.